### PR TITLE
Add ErrorStream Facade

### PR DESCRIPTION
**Pull request by Tucker-Eric**

### CODE REVIEW

**Comment by kevinhamilton** [comment]
Hey Eric- thanks for the PR. Can you update the readme to fix the typo. It should read "add". Then I'll get this merged and the site updated by the end of the night.

File: README.md [hunk 1]
```diff
@@ -15,13 +15,20 @@ Next, register the service provider in the config/app.php file.
      ErrorStream\ErrorStream\ErrorStreamServiceProvider::class,
 ]
 ```
+Then ass the Facade to the aliases array in the config/app.php file.
```
Comment (kevinhamilton) [inline README.md]: this is the typo i was talking about.

Comment (Tucker-Eric) [inline README.md]: Oh gosh, didn't even catch that.  Updated now ;-)

**Comment by kevinhamilton** [comment]
This change is now available for you under branch 3.1. Thanks for contributing!

**Pull request merged**